app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
@Fryguy will advanced_settings show on the Node/Provider page? Will I be able to use them in conditions?